upup/pkg/kutil/drain.go

```diff
@@ -0,0 +1,674 @@
+/*
```

Comment:
Why are we adding more code to kutil? This package seems a bit misc'y enough as it is.. wondering if there is a better home for this.